nest-demo/docker/docker-compose.yml

```
@@ -0,0 +1,129 @@
+    image: grafana/loki:3.5.0
+    container_name: loki
+    restart: unless-stopped
+    env_file:
+      - .env.loki
+    command: -config.file=/etc/loki/loki.yaml
+    user: ":"
+    ports:
+       - "3100:3100"
+       - "9096:9096"
+    healthcheck:
+        test: wget -q --tries=1 -O- http://localhost:3100/ready 
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)